### ISSUE TITLE
[Sourcing]fdp-table expand all / collapse all need tooltips , aria-labels

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9551]
#### Is this a bug, enhancement, or feature request?
Bug
#### Briefly describe your proposal.
Expand all / collapse all buttons for fdp-table needs to have tooltips and aria-labels.
Coz just looking at the icon buttons , user may not be able to understand the purpose of the buttons

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 15
core / platform: 0.39.1-rc-5

**Comment by fkolar** [comment]
We need to be clear that Button does not have tooltip, they have only Aria-labels and this is already provided by fundamental  core library. We are discussing only Icon Buttons here. 

The exapnd /collapse button  ICONs they do miss toolings. True. 

@droshev , @mikerodonnell89  It seems this needs its not only table but the root cause is core button, when glyph is used we should provide a toolip.

We can do this in the way: 

```html
<fd-icon glyph="slim-arrow-down" *ngIf="fdMenu"></fd-icon>

<ng-template #iconTemplate>
    <fd-icon *ngIf="glyph" [glyph]="glyph"></fd-icon>

    <ng-content></ng-content>
</ng-template>

```


@I543348 , When IconButton (label + glyph) - no tooltip, when only icon  - provide a tooltip. This is similar to SAPUI5
